core/os/keyboard.cpp

```diff
@@ -63,12 +63,15 @@ static const _KeyCodeText _keycodes[] = {
 	{Key::CTRL                  ,"Ctrl"},
 #if defined(MACOS_ENABLED)
 	{Key::META                  ,"Command"},
+    {Key::CMD_OR_CTRL           ,"Command"},
```

Comment:
The indentation is wrong, should use a tab.

I'm surprised the CI checks don't complain about it.

Reply:
don't know why it put spaces nor why it didn't complain but fixed it